client/templates/juror-management/_partials/juror-update-options.njk

```diff
@@ -0,0 +1,126 @@
+{% endif %}
+
+{% if jurorStatus === "Excused" and owner === "400" %}
+  {% set updateOptions = [responded, deferral, excusal, disqualify, postpone, deceased] %}
+{% endif %}
+
+{% if jurorStatus === "Disqualified" and owner === "400" %}
+  {% set updateOptions = [responded, deferral, excusal, postpone, deceased] %}
+{% endif %}
+
+{% if jurorStatus === "Excused" and excusalCode === "D" and owner === "400" %}
+  {% set updateOptions = [responded, deferral, excusal, postpone, disqualify] %}
+{% endif %}
+
+{% if jurorStatus === "Deferred" and owner === "400" %}
+  {% set updateOptions = [deferral, excusal, postpone, disqualify, deceased] %}
+{% endif %}
+
+
+
+{# court owned options #}
+{% if jurorStatus === "Summoned" and owner !== "400" %}
+  {% set updateOptions = [responded, deferral, excusal, disqualify, postpone, deceased, undeliverable] %}
+{% endif %}
+
+{# responded and no attendances #}
+{% if jurorStatus === "Responded" and attendances === 0 and owner !== "400" %}
+  {% set updateOptions = [deferral, excusal, disqualify, postpone, deceased, complete, failedToAttend] %}
+{% endif %}
+
+{# responded and attendances #}
+{% if jurorStatus === "Responded" and attendances > 0 and owner !== "400" %}
+  {% set updateOptions = [disqualify, deceased, complete] %}
+{% endif %}
+
+{% if jurorStatus === "Excused" and owner !== "400" %}
+  {% set updateOptions = [responded, deferral, excusal, disqualify, postpone, deceased] %}
+{% endif %}
+
+{% if jurorStatus === "Disqualified" and owner !== "400" %}
+  {% set updateOptions = [responded, deferral, excusal, postpone, deceased, complete] %}
+{% endif %}
+
+{# deceased and no attendances #}
+{% if jurorStatus === "Excused" and excusalCode === "D" and attendances === 0 and owner !== "400" %}
+  {% set updateOptions = [responded, deferral, excusal, disqualify, postpone, complete] %}
+{% endif %}
+
+{# deceased and attendances #}
+{% if jurorStatus === "Excused" and excusalCode === "D" and attendances > 0 and owner !== "400" %}
+  {% set updateOptions = [responded, disqualify, complete] %}
+{% endif %}
+
+{% if jurorStatus === "Deferred" and owner !== "400" %}
+  {% set updateOptions = [deferral, excusal, disqualify, postpone, deceased] %}
+{% endif %}
+
+{# responded and juror is on call and no attendances #}
+{% if jurorStatus === "Responded" and attendances === 0 and onCall and owner !== "400" %}
+  {% set updateOptions = [deferral, excusal, disqualify, postpone, deceased, complete, failedToAttend] %}
+{% endif %}
+
+{# responded an juror is on call and has attendances #}
+{% if jurorStatus === "Responded" and attendances > 0 and onCall and owner !== "400" %}
+  {% set updateOptions = [disqualify, deceased, complete] %}
+{% endif %}
+
+{% if jurorStatus === "FailedToAttend" and owner !== "400" and isSJO %}
+  {% set updateOptions = [undoFailedToAttend] %}
+{% endif %}
+
+{# uncomplete is not implemented through the juror record #}
+{# {% if jurorStatus === "Completed" and owner !== "400" and isSJO %}
```

Comment:
Is this worth getting rid of it if shouldn't be here then?

Reply:
you make a good point that I can't say no to :-)